netlify.toml

```
@@ -6,9 +6,9 @@
   package = "@netlify/plugin-lighthouse"
 
   [plugins.inputs.thresholds]
-    performance = 0.82
+    performance = 0.65
```

I think `0.65` is too low, try increasing it into `0.75` instead, since the last build was having the `0.75` score.
```suggestion
    performance = 0.75
```